src/libsystemd/sd-bus/bus-message.c

```diff
@@ -4667,3 +4667,26 @@ _public_ int sd_bus_message_sensitive(sd_bus_message *m) {
         m->sensitive = true;
         return 0;
 }
+
+char** bus_message_make_log_fields(sd_bus_message *m) {
+        _cleanup_strv_free_ char **strv = NULL;
+
+        assert(m);
+
+        (void) strv_extend_join(&strv, "DBUS_MESSAGE_TYPE=", bus_message_type_to_string(m->header->type));
```

Comment:
I think strv_extend_join() shiuld really be called strv_append_assignment(), and then insert the `=` on its own